app/migrate.go

```diff
@@ -42,6 +42,7 @@ const (
 	FlagDestPath         = "dest-path"
 	FlagDestHostOverride = "dest-host-override"
 	FlagLBSvcTimeout     = "lbsvc-timeout"
+	FlagNodePortPort     = "nodeport-port"
```

Comment:
I'm wondering if we need this flag at all - we allow specifying any overrides to the helm chart values - see examples 4 and 5 here: https://github.com/utkuozdemir/pv-migrate/blob/master/USAGE.md.

It would make things way simpler - the whole reason I made this a helm chart (it wasn't a chart at the earlier times of this project) was to allow such customizations without a need to introduce new flags all the time.

Can you please give it a try? `--helm-set sshd.service.nodePort=32123` basically. If it works, we won't need this flag and all the handling around it.